Container/CSS/button.css

```diff
@@ -0,0 +1,84 @@
+@import url(/Container/CSS/style.css);
+
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: content-box;
+}
+
+.btn{
+    display: flex;
+    align-items: center;
+    gap: 1rem;
+}
+
+.btn-style{
+    padding: 0.4rem 1.2rem;
+    border-radius: 0.2rem;
+    font-family: var(--font-primary);
+    font-size: 1rem;
+    font-weight: 500;
+    outline: 1px solid black;
+    cursor: pointer;
+    margin: 0.5rem;
+}
+.primary-btn{
+    color: var(--text-color);
+    background: var(--tertiary-color);
+}
+.danger-btn{
+    color: var(--text-color);
+    background: var(--danger-alert);
+}
+.warning-btn{
+    color: var(--text-color);
+    background: var(--warning-alert);
+}
```

Comment:
since the text-color is repeating we can try writing it under a common class

Reply:
yes, ill figure out how to do that and do it.